controllers/apps/operations/stop.go

```diff
@@ -59,17 +55,19 @@ func (stop StopOpsHandler) ActionStartedCondition(reqCtx intctrlutil.RequestCtx,
 // Action modifies Cluster.spec.components[*].replicas from the opsRequest
 func (stop StopOpsHandler) Action(reqCtx intctrlutil.RequestCtx, cli client.Client, opsRes *OpsResource) error {
 	var (
-		componentReplicasMap = map[string]int32{}
-		cluster              = opsRes.Cluster
+		cluster = opsRes.Cluster
 	)
+
 	// if the cluster is already stopping or stopped, return
 	if slices.Contains([]appsv1alpha1.ClusterPhase{appsv1alpha1.StoppedClusterPhase,
 		appsv1alpha1.StoppingClusterPhase}, opsRes.Cluster.Status.Phase) {
 		return nil
 	}
-	if _, ok := cluster.Annotations[constant.SnapShotForStartAnnotationKey]; ok {
-		return fmt.Errorf("wait for the cluster to start before continuing to stop the cluster")
-	}
+	// TODO
+	// if _, ok := cluster.Annotations[constant.SnapShotForStartAnnotationKey]; ok {
+	//	return fmt.Errorf("wait for the cluster to start before continuing to stop the cluster")
+	// }
```

Comment:
Clusters in a stopped state that were created using KB 0.9.x should also be migrated to KB Cluster v1.0. Therefore, this commented code segment may be unnecessary?